src/hooks/use-assistant.ts

```diff
@@ -94,6 +98,23 @@ export const useAssistant = ( instanceId: string ) => {
 		[ instanceId ]
 	);
 
+	const updateFailedMessage = useCallback(
```

Comment:
[ Proposal ] What if we update the name to `markMessageAsFailed`? Isn't that a bit more descriptive?

Reply:
Hey hey I did not notice the comment before I merged so I added small PR for this: https://github.com/Automattic/studio/pull/322

I think it is a good idea, especially since we have another function for updating the message already.